search.html

```diff
@@ -31,22 +31,38 @@
     <div class="container">
         <div class="row">
             <!-- This is the wrapper for the aside and main -->
+            
             <aside class="col s3" id="container-aside">
                 <div id="container-search">
                     <form>
                         <input type="text" id="search-input" name="search-input" placeholder="Movie Name">
                     </form>
-                    <button id="btn-search" class="waves-effect waves-light btn">Search</button>
+                    <button id="btn-search" class="btn btn-search"><i title="Search" class="material-icons">search</i></button>
```

Comment:
Can we put the waves-effect and waves-light classes back onto the button here on line 40?